ApplicationLibCode/ProjectDataModel/Summary/RimSummaryRegressionAnalysisCurve.cpp

```diff
@@ -382,3 +383,94 @@ QString RimSummaryRegressionAnalysisCurve::generateRegressionText( const regress
     // TODO: Display more parameters here.
     return "";
 }
+
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RimSummaryRegressionAnalysisCurve::appendTimeSteps( std::vector<time_t>& timeSteps, const std::set<QDateTime>& moreTimeSteps )
```

Comment:
Maybe rename to destinationTimeSteps and sourceTimeSteps?

Reply:
Fixed.